### PR TITLE
Towards agent parity

### PR DESCRIPTION
**Pull request by ticky**
- [x] Show Job link
- [x] Show build status icon
- [x] Add agent installation docs

### CODE REVIEW

**Comment by ticky** [comment]
I’m really excited for this!!!

animated 2016-11-01 at 18 44 42

**Comment by keithpitt** [comment]
Oh awesome! I am also very excited by this...

**Comment by toolmantim** [comment]
SO CLOSE!

**Comment by sj26** [comment]
Excitement! ⚡️

File: app/components/agent/Index/quickStart.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+              key={index}
+              href="#"
+              className={classNames(
+                'inline-block blue hover-navy text-decoration-none border rounded m1 p1',
+                {
+                  'border-white': index !== this.state.selectedGuide,
+                  'border-gray': index === this.state.selectedGuide
+                }
+              )}
+              onClick={this.handleSelectedGuideChange.bind(this, index)} // eslint-disable-line react/jsx-no-bind
```
Comment (keithpitt) [inline app/components/agent/Index/quickStart.js]: Hrm...I'm probably missing something, but why did you need to disable eslint here?

File: app/components/agent/Index/quickStart.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+              style: { fontFamily: 'Monaco, Consolas, monospace', fontSize: '.9em' }
+            },
+            'a': {
+              className: 'blue hover-navy text-decoration-none hover-underline'
+            },
+            'pre': {
+              className: 'border border-gray rounded bg-silver overflow-auto p2 monospace'
+            }
+          }}
+          token={this.props.organization.agentTokens.edges[0].node.token}
```
Comment (keithpitt) [inline app/components/agent/Index/quickStart.js]: I think this could break if the agentTokens haven't been loaded yet?

File: app/components/agent/Index/quickStart.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        />
+      );
+    }
+  }
+
+  render() {
+    return (
+      <Panel className="mb3">
+        <Panel.Header><center>Agent Quick Start Guides</center></Panel.Header>
```
Comment (keithpitt) [inline app/components/agent/Index/quickStart.js]: `<Panel.Header className="center">` perhaps to be consistent with `center` usage elsewhere?

File: app/components/agent/Index/row.js [hunk 4]
```diff
@@ -22,30 +28,69 @@ class AgentRow extends React.Component {
...
       <Panel.Row>
         <div className="flex">
-          <div className="pr3 pt1">
-            <div className={iconClassName} style={{ width: 12, height: 12 }} />
+          <div
+            className="pr3 pt1"
+            title={getLabelForConnectionState(agent.connectionState)}
+          >
+            {
+              agent.job
```
Comment (keithpitt) [inline app/components/agent/Index/row.js]: This bit of JS is breaking my brain a little :P

Perhaps it's better suited to another function in this component?

File: app/components/agent/Index/row.js [hunk 5]
```diff
@@ -22,30 +28,69 @@ class AgentRow extends React.Component {
...
+                </div>
+                <small className="dark-gray">{metaDataContent}</small>
+              </div>
+              <div className="right-align">
+                <div className="black">v{agent.version}</div>
+                <small className="dark-gray">{agent.hostname}</small>
+              </div>
+            </div>
+            {
+              agent.job
```
Comment (keithpitt) [inline app/components/agent/Index/row.js]: This one should probably be a function as well I think!

File: app/components/agent/Index/quickStart.js (hunk 1 above)
Comment (ticky) [inline app/components/agent/Index/quickStart.js]: ESLint complains about the use of `bind`, but I don’t see a cleaner way to do this.

File: app/components/agent/Index/quickStart.js (hunk 3 above)
Comment (ticky) [inline app/components/agent/Index/quickStart.js]: `Panel.Header` doesn’t pass `className` through, whereas `Panel` does. I’ll fix it up!

File: app/components/agent/Index/quickStart.js (hunk 1 above)
Comment (keithpitt) [inline app/components/agent/Index/quickStart.js]: Ah yeah, this is why we do this '=' thingy here:

https://github.com/buildkite/frontend/blob/master/app/components/organization/Pipeline/index.js#L144

File: app/components/docs/03-redhat-or-centos.mdx [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: RH/CentOS
+emoji: ':redhat:'
```
Comment (keithpitt) [inline app/components/docs/03-redhat-or-centos.mdx]: This is cool tech

File: app/components/agent/Index/quickStart.js (hunk 1 above)
Comment (keithpitt) [inline app/components/agent/Index/quickStart.js]: Oh ok!

**Comment by keithpitt** [comment]
🚀
